### PR TITLE
Update internet-facing-devices.md

### LINKED ISSUE #12360: Internet facing devices also applies to Microsoft Defender for Business
[Enter feedback here]
The applies to section should also include Microsoft Defender for Business (The SMB version based on Microsoft Defender for Endpoint P2)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 32c069f3-88bb-b506-4b3a-da4b628f2f84
* Version Independent ID: 32c069f3-88bb-b506-4b3a-da4b628f2f84
* Content: [Identify internet-facing devices in Microsoft Defender for Endpoint](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/internet-facing-devices?view=o365-worldwide&source=docs)
* Content Source: [microsoft-365/security/defender-endpoint/internet-facing-devices.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/internet-facing-devices.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @siosulli
* Microsoft Alias: **siosulli**

### PR DESCRIPTION
**Pull request by msbemba**
Added defender for business to the applies to section

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12360

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/internet-facing-devices.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ ms.date: 03/7/2023
 - [Microsoft Defender for Endpoint Plan 1](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 - [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 - [Microsoft 365 Defender](https://go.microsoft.com/fwlink/?linkid=2118804)
+- [Microsoft Defender for Business](/microsoft-365/security/defender-business/)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/internet-facing-devices.md]:
```suggestion
- [Microsoft Defender for Business](/microsoft-365/security/defender-business)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**